tests/test_utils.py

```diff
@@ -49,8 +49,16 @@ def test_list_exchanges(capsys):
     assert re.search(r"^bittrex$", captured.out, re.MULTILINE)
 
 
-def test_list_timeframes(capsys):
-
+def test_list_timeframes(mocker, capsys):
+
+    api_mock = MagicMock()
+    api_mock.timeframes = {'1m': '1m',
```

Comment:
I guess, keys and vals should differ in this dict (arbitrary vals can be used like `'1m': '1m one', '5m': '5m two'`) to be sure in the test that the keys and not values are used by the bot...

Reply:
well yes - but i copied most of it from here ... :D 

https://github.com/ccxt/ccxt/blob/ea91f5f738f5338e5087677c0b619931831c1caa/python/ccxt/binance.py#L50-L65

now we have both versions - binance and bittrex ...